app/src/main/res/layout/activity_main.xml

```diff
@@ -12,47 +12,56 @@
     tools:context="com.codeforireland.vacanthome.MainActivity">
 
     <ImageView
+        android:id="@+id/logo"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:id="@+id/logo"
-        android:src="@drawable/logo1"
         android:layout_centerHorizontal="true"
-        ></ImageView>
+        android:layout_marginEnd="@dimen/activity_horizontal_margin"
+        android:layout_marginLeft="@dimen/activity_horizontal_margin"
+        android:layout_marginRight="@dimen/activity_horizontal_margin"
+        android:layout_marginStart="@dimen/activity_horizontal_margin"
+        android:layout_marginTop="@dimen/activity_vertical_margin"
+        android:src="@drawable/logo1"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"/>
 
     <Button
-        android:text="Log a vacant property"
-        android:layout_width="200dip"
-        android:layout_height="wrap_content"
-        android:layout_centerHorizontal="true"
-        android:layout_below="@+id/textView"
-        android:layout_marginTop="25dp"
         android:id="@+id/button"
-        android:background="#28b3ed"
-        android:textColor="#ffffff"
-        />
+        style="@style/button_green"
+        android:layout_marginEnd="@dimen/activity_horizontal_margin"
+        android:layout_marginLeft="@dimen/activity_horizontal_margin"
+        android:layout_marginRight="@dimen/activity_horizontal_margin"
+        android:layout_marginStart="@dimen/activity_horizontal_margin"
+        android:layout_marginTop="@dimen/activity_vertical_margin"
+        android:text="@string/btn_start_log_vacant_property"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/logo" />
 
     <TextView
-        android:text="Thank You for submitting this information, from the vacant homes team and Code For Ireland"
-        android:textAlignment="center"
+        android:id="@+id/textView"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:layout_alignLeft="@+id/button"
-        android:layout_alignStart="@+id/button"
         android:layout_below="@+id/logo"
-        android:id="@+id/textView"
-        android:layout_alignRight="@+id/button"
-        android:layout_alignEnd="@+id/button"
+        android:layout_marginTop="@dimen/activity_vertical_margin"
+        android:text="@string/title_main_thank_you_for_submitting_this_information_from_the_vacant_homes_team_and_code_for_ireland"
```

Comment:
Can we shorten this name? It's very long

Reply:
Done